.github/workflows/ci-cd.yml

```diff
@@ -506,33 +505,16 @@ jobs:
         - macos-latest
         - windows-latest
         python-version:
+        - 3.13
```

Comment:
Put highest+lowesr+pypy at the beginning of this list. This is to prioritize responsiveness of the most important jobs. If these pass, the entire range is likely to pass. But it they fail, it's convenient to learn about this a little earlier. So I sort them by putting the edge cases first and then the rest is high to low.